packages/react/src/experimental/SelectPanel2/SelectPanel.tsx

```diff
@@ -380,26 +340,13 @@ const SelectPanelHeader: React.FC<React.PropsWithChildren & {onBack?: () => void
   const {title, description, panelId, onCancel, onClearSelection} = React.useContext(SelectPanelContext)
 
   return (
-    <Box
-      sx={{
-        display: 'flex',
-        flexDirection: 'column',
-        // gap: 2,
-        padding: 2,
-        borderBottom: '1px solid',
-        borderColor: 'border.default',
-      }}
-      {...props}
-    >
-      <Box
-        sx={{
-          display: 'flex',
-          justifyContent: 'space-between',
-          alignItems: description ? 'start' : 'center',
-          marginBottom: slots.searchInput ? 2 : 0,
-        }}
+    <div className={classes.Header} {...props}>
```

Comment:
Should we support className here? my concern would be it being overriden by {...props} if set by the consumer.